example/build_cmake_installed_kk_as_language/CMakeLists.txt

```diff
@@ -4,14 +4,21 @@ cmake_minimum_required(VERSION 3.19)
 
 # Projects can safely mix languages - must have C++ support
 # Kokkos flags will only apply to C++ files
-project(Example CXX Fortran CUDA)
+project(Example CXX Fortran)
 
 # Look for an installed Kokkos
-find_package(Kokkos COMPONENTS separable_compilation)
-set_source_files_properties(cmake_example.cpp PROPERTIES LANGUAGE CUDA)
+find_package(Kokkos)
+enable_language(${Kokkos_COMPILE_LANGUAGE})
+
+#the language has to be set on the files
+set_source_files_properties(cmake_example.cpp PROPERTIES LANGUAGE ${Kokkos_COMPILE_LANGUAGE})
 add_executable(example cmake_example.cpp bar.cpp foo.f)
 
-# This is the only thing required to set up compiler/linker flags
+# The architectures and std have to be set on the target
+set_target_properties(
+  example PROPERTIES ${Kokkos_COMPILE_LANGUAGE}_ARCHITECTURES ${Kokkos_${Kokkos_COMPILE_LANGUAGE}_ARCHITECTURES}
```

Comment:
What happens with cmake when you set a property that does not exists?
As I understand it, there is no such thing as a CXX_ARCHITECTURES property and I am somewhat surprised it does not error out that the value argument is missing (trailing arg does not exists and presumably does not exists)

Reply:
It does
```
CMake Error at CMakeLists.txt:18 (set_target_properties):
  set_target_properties called with incorrect number of arguments.
```
In my understanding this example is only meant to be used (at this point) when the `HIP` or `Cuda` backend is enabled.

Reply:
hmm `set_properties` just silently does nothing for CXX as language ... but not sure if we want to it this way